src/tiktokbot.ts

```diff
@@ -0,0 +1,283 @@
+        secondary.map((obj: { [key: string]: any }) => Object.keys(obj).map((k: string) => main[k] = obj[k]))
+        return main
+    }
+
+    /**
+     * Make a millisecond time number readable.
+     * 
+     * @param [ms] The ms time to convert.
+     * @return string
+     * 
+     * @example
+     * <TikTok>.timeify(60 * 1000): "1 min(s)" (string)
+     */
+    private timeify = (ms: number) => {
+        // Declare variables.
+        let sec: number = Math.round(ms / 1000)
+        let min: number = Math.floor(sec / 60)
+        let hrs: number = 0
+        const final: string[] = []
+
+        // Complete logic.
+        if (min >= 60) { (hrs = Math.floor(min / 60)) ? final.push(`${hrs} hr(s)`) : null }
+        { (min -= hrs * 60) ? final.push(`${min} min(s)`) : null }
+        { (sec = Math.floor(sec % 60)) ? final.push(`${sec} sec(s)`) : null }
+        return final.join(', ')
+    }
+
+    /**
+     * <discord.js> Client event: "ready".
+     * 
+     * @param [] None required.
+     * @return null
+     * 
+     * @example 
+     * <TikTok>.ready(): null
+     */
+    private ready = () => console.log(`Connected to ${this.user?.tag ?? 'Client'}.`)
+
+    /**
+     * <discord.js> Client event: "messageCreate".
+     * 
+     * @param {import('discord.js').Message} [message] <discord.js> Message object.
+     * @return null
+     * 
+     * @example 
+     * <TikTok>.messageCreate(<Message>): null
+     */
+    private async messageCreate(message: Message) {
+
+        // Check for bot message or DM message.
+        if (message.author?.bot || message.channel?.type?.is('DM')) return
+
+        // Check for commands.
+        if (message.content.match(new RegExp(`^(<@!?${this.user?.id}>.*)`, 'gm'))?.some((e: string) => e)) {
+
+            // Set custom message properties.
+            { (message as MessageExtender).args = message.content.split(' ').slice(1) }
+            { (message as MessageExtender).command = (message as MessageExtender).args.shift() ?? '' }
+
+            if ((message as MessageExtender).command.is('invite')) {
+
+                // Invite command.
+                const url: string = `https://discord.com/api/oauth2/authorize?client_id=${this.user?.id}&permissions=274878032960&scope=bot`
+                const embeds: object[] = [{
+                    color,
+                    author: {
+                        name: this.user?.tag,
+                        iconURL: this.user?.avatarURL({ format: 'png', dynamic: true, size: 128 })
+                    },
+                    title: 'Click me!',
+                    url,
+                    description: `Raw URL: [${url}](${url})`
+
+                }]
+
+                // Reply to author.
+                message.reply(this.mergify(this.reply, { embeds }))
+            } else {
+
+                // If bot is mentioned but no command is specified.
+                const embeds: object[] = [{
```

Comment:
```suggestion
                const embeds: MessageEmbed[] = [{
```

Reply:
I will update to `MessageEmbed[] | object[]` because keeping it just `MessageEmbed[]` causes this error:
```bash
Type '{ color: number; author: { name: string; iconURL: string; }; title: string; description: string; fields: { name: string; value: string; inline: false; }[]; }' is missing the following properties from type 'MessageEmbed': _fieldEquals, createdAt, footer, hexColor, and 23 more.ts(2740)
```